src/Decoder.ts

```diff
@@ -44,7 +44,9 @@ try {
   EMPTY_VIEW.getInt8(0);
 } catch (e) {
   if (!(e instanceof RangeError)) {
-    throw new Error("This module is not supported in the current JavaScript engine because DataView does not throw RangeError on out-of-bounds access");
+    throw new Error(
+      "This module is not supported in the current JavaScript engine because DataView does not throw RangeError on out-of-bounds access",
+    );
```

Comment:
Can you disable and revert any IDE linting changes? That'll make it easier to merge any upstream changes into this repo :)

Reply:
(see also extra line break below)